docs-conceptual/azps-2.0.0/migrate-az-1.0.0.md

```diff
@@ -42,146 +47,181 @@ This document describes the changes between the 6.x versions of AzureRM and Az v
-```powershell
+
+Scripts that use module-qualified cmdlet invocations, such as:
+
+```azurepowershell-interactive
 AzureRM.Compute\Get-AzureRmVM
 ```
 
-Should be changed to use the new module and cmdlet names
-```powershell
+Must be changed to use the new module and cmdlet names:
+
+```azurepowershell-interactive
 Az.Compute\Get-AzVM
 ```
 
-### Migrating Module Manifest Dependencies
-Modules that express dependencies on AzureRM modules through a module manifest (.psd1) file will need to updated the module names in their 'RequiredModules' section
+### Migrating module manifest dependencies
+
+Modules that express dependencies on AzureRM modules through a module manifest (.psd1) file will need to updated the module names in their `RequiredModules` section:
 
-```powershell
+```azurepowershell-interactive
 RequiredModules = @(@{ModuleName="AzureRM.Profile"; ModuleVersion="5.8.2"})
 ```
 
-Should be changed to
+Must be changed to:
 
-```powershell
+```azurepowershell-interactive
 RequiredModules = @(@{ModuleName="Az.Profile"; ModuleVersion="1.0.0"})
 ```
 
 ### Removed modules
+
+The following modules have been removed:
+
 - `AzureRM.Backup`
 - `AzureRM.Compute.ManagedService`
 - `AzureRM.Scheduler`
 
-The tooling for these services are no longer actively supported.  Customers are encouraged to move to alternative services as soon as it is convenient.
+The tools for these services are no longer actively supported.  Customers are encouraged to move to alternative services as soon as it is convenient.
 
 ### Windows PowerShell 5.1 and .NET 4.7.2
-- Using Az with Windows PowerShell 5.1 requires the installation of .NET 4.7.2. However, using Az with PowerShell Core does not require .NET 4.7.2. 
+
+Using Az with Windows PowerShell 5.1 requires the installation of .NET 4.7.2. However, using Az with PowerShell Core does not require .NET 4.7.2. 
 
 ### Temporary removal of User login using PSCredential
-- Due to changes in the authentication flow for .NET Standard, we are temporarily removing user login via PSCredential. This capability will be re-introduced in the 1/15/2019 release for Windows PowerShell 5.1. This is discussed in detail in [this issue.](https://github.com/Azure/azure-powershell/issues/7430)
 
-### Default Device Code login instead of Web Browser prompt
-- Due to changes in the authentication flow for .NET Standard, we are using device login as the default login flow during interactive login. Web browser based login will be re-introduced for Windows PowerShell 5.1 as the default in the 1/15/2019 release. At that time, users will be able to choose device login using a Switch parameter.
+Due to changes in the authentication flow for .NET Standard, we are temporarily removing user login via PSCredential. This capability will be re-introduced in the 1/15/2019 release for Windows PowerShell 5.1. This is discussed in detail in [this GitHub issue.](https://github.com/Azure/azure-powershell/issues/7430)
```

Comment:
We are past this date. Is this still a problem? I am not aware of 1/15/2019 release of PowerShell 5.1. Or do you mean the release of Az PowerShell and not Windows PowerShell 5.1?

Reply:
@dcaro I was wondering this myself. The github issues are a labrynth here, I clicked through something like 4 layers of them and still didn't find out whether or not this has been fixed.